src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/PremisesUtils.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.util
+
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.DateCapacity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.PremisesEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.PremisesService
+import java.time.LocalDate
+
+fun getDateCapacities(premises: PremisesEntity, premisesService: PremisesService): List<DateCapacity> {
```

Comment:
Does this belong in the previous commit?

Reply:
Ah, gotcha. This gets moved back into the service in the next commit doesn't it? Probably worth deleting it in that case 👍